### PR TITLE
pipeline for depth data

### PR DESCRIPTION
**Pull request by lawinslow**
@jread-usgs @mhines-usgs want to take a look?

### CODE REVIEW

File: remake_depth.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  - scripts/encryption_helpers.R
+  - scripts/depth_link_jacobson.R
+  - scripts/depth_summarize.R
+  - scripts/depth_link_lagos.R
+
+
+targets:
+  all: 
+    depends: 
+      - data/depth_data_linked/all_depths.edt
```
Comment (jordansread) [inline remake_depth.yml:29]: does this need the key locally to work? is it a env.var or in the internal yaml?

File: scripts/depthNHD.R [hunk 2]
```diff
@@ -1,7 +1,8 @@
-library(rgdal)
+#library(rgdal)
+
 
 #get nhd layer
-nhd <- readOGR(dsn = paste0(getwd(),"/data"), layer="NHDWaterbody")
+nhd <- readOGR(dsn = file.path(getwd(),"data", "NHD_shape_large"), layer="NHDWaterbody")
```
Comment (jordansread) [inline scripts/depthNHD.R:5]: can we assume all `make` is running from the project root?

**Comment by jordansread** [comment]
:eyes: good to me

File: remake_depth.yml (hunk 1 above)
Comment (lawinslow) [inline remake_depth.yml:29]: It needs the key to work. This uses the internal yaml

File: scripts/depthNHD.R (hunk 2 above)
Comment (lawinslow) [inline scripts/depthNHD.R:5]: Yeah, that's just legacy stuff I didn't pull out.

**Pull request merged**